### ISSUE TITLE
Error: Attempt to index global 'L' (a nil value)

### ISSUE THREAD

**Issue by Vysci** [Vysci/LFG-Bulletin-Board#116]
Error reporting by "TeacherPlaysTV"

```
Message: ...rface\AddOns\LFGBulletinBoard\GroupBulletinBoard.lua:334: attempt to index global 'L' (a nil value)
Time: Thu Jul 8 10:30:01 2021
Count: 1
Stack: ...rface\AddOns\LFGBulletinBoard\GroupBulletinBoard.lua:334: attempt to index global 'L' (a nil value)
[string "=[C]"]: ?
[string "@Interface\AddOns\LFGBulletinBoard\GroupBulletinBoard.lua"]:334: in function <...rface\AddOns\LFGBulletinBoard\GroupBulletinBoard.lua:316>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: in function `SetUnit'
[string "@Interface\FrameXML\UnitFrame.lua"]:284: in function `UnitFrame_UpdateTooltip'
[string "@Interface\FrameXML\UnitFrame.lua"]:270: in function `UnitFrame_OnEnter'
[string "*:OnEnter"]:1: in function <[string "*:OnEnter"]:1>
```

```

Message: ...rface\AddOns\LFGBulletinBoard\GroupBulletinBoard.lua:334: attempt to index global 'L' (a nil value)
Time: Thu Jul 8 14:49:44 2021
Count: 1
Stack: ...rface\AddOns\LFGBulletinBoard\GroupBulletinBoard.lua:334: attempt to index global 'L' (a nil value)
[string "=[C]"]: ?
[string "@Interface\AddOns\LFGBulletinBoard\GroupBulletinBoard.lua"]:334: in function <...rface\AddOns\LFGBulletinBoard\GroupBulletinBoard.lua:316>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
```

**Comment by Stanzilla** [comment]
Yep, same here:

```
623x ...rfaceLFGBulletinBoard\GroupBulletinBoard.lua:334: attempt to index global 'L' (a nil value)
[string "@LFGBulletinBoard\GroupBulletinBoard.lua"]:334: in function <...rfaceLFGBulletinBoard\GroupBulletinBoard.lua:316>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
```

**Comment by rpillala** [comment]
For me, this error occurs when I mouse over someone I have recently grouped with.  

Message: ...rface\AddOns\LFGBulletinBoard\GroupBulletinBoard.lua:334: attempt to index global 'L' (a nil value)
Time: Mon Jul 12 19:46:47 2021
Count: 2
Stack: ...rface\AddOns\LFGBulletinBoard\GroupBulletinBoard.lua:334: attempt to index global 'L' (a nil value)
[string "=(tail call)"]: ?
[string "=[C]"]: ?
[string "@Interface\AddOns\LFGBulletinBoard\GroupBulletinBoard.lua"]:334: in function <...rface\AddOns\LFGBulletinBoard\GroupBulletinBoard.lua:316>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: ?

Locals: (*temporary) = <function> defined =[C]:-1

**Comment by Vysci** [comment]
I’ll have a fix for this ~ tomorrow

**Comment by Vysci** [comment]
Should be fixed in the latest version 2.60 (up on curseforge). I'll update release notes when I have access to my regular pc
